app/statistics/server/functions/get.js

```diff
@@ -154,5 +156,34 @@ statistics.get = function _getStatistics() {
 	statistics.uniqueOSOfYesterday = Sessions.getUniqueOSOfYesterday();
 	statistics.uniqueOSOfLastMonth = Sessions.getUniqueOSOfLastMonth();
 
+	if (Apps.isEnabled()) {
+		const installedApps = Apps.getManager().get();
+		const activeApps = Apps.getManager().get({ enabled: true });
+		statistics.apps = {
+			totalInstalled: installedApps.length,
+			totalActive: activeApps.length,
+			installedAppsList: installedApps.map((app) => ({
+				id: app.getID(),
+				name: app.getName(),
+				status: app.getStatus(),
+			})),
+		};
+	}
+
+	const integrations = Integrations.find().fetch();
+
+	statistics.integrations = {
+		totalIntegrations: integrations.length,
+		totalActiveIntegrations: integrations.filter((integration) => integration.enabled === true).length,
+		totalIncoming: integrations.filter((integration) => integration.type === 'webhook-incoming').length,
+		totalOutgoing: integrations.filter((integration) => integration.type === 'webhook-outgoing').length,
+		integrationList: integrations.map((integration) => ({
```

Comment:
same as with apps, why do we need this data? specially the integration name?

Reply:
I do think this is only mildly useful, but do not oppose it. If you think there is a reason _not_ to send this data, I'm all ears :)

Also, as this was @geekgonecrazy input, he might have something to add here :)

Reply:
Same with the other.  I think count and maybe how many active vs inactive would be sufficient